cypress/Dockerfile

```diff
@@ -50,15 +44,44 @@ COPY ../src/tools/larft/config/ src/tools/larft/config/
 
 # Enable creation of screenshots/videos
 RUN useradd -M -d /hmda-frontend -s /bin/bash hmda_cypress_user && \
-  chown -R hmda_cypress_user:hmda_cypress_user /hmda-frontend/
+chown -R hmda_cypress_user:hmda_cypress_user /hmda-frontend/
 USER hmda_cypress_user
+# USER root
+
+# Uninstall prebuilt cypress if exists
+# RUN npm uninstall cypress -g
+# RUN rm -rf /root/.cache/Cypress/*
 
 # Install Cypress + supporting libraries
 COPY cypress/package.json ./
 RUN yarn install
 
-# Cleanup vulnerabilities (as hmda_cypress_user)
-RUN rm -rf /hmda-frontend/node_modules/public-encrypt/test/*
-RUN rm -rf /hmda-frontend/.cache/Cypress/**/Cypress/resources/app/node_modules/public-encrypt/test/*
-RUN rm -rf /hmda-frontend/.cache/Cypress/**/Cypress/resources/app/node_modules/http-proxy/test/*
-RUN rm -rf /hmda-frontend/.cache/Cypress/**/Cypress/resources/app/node_modules/lazystream/secret
\ No newline at end of file
+# Download fixtures
+RUN mkdir -p cypress/downloads
+RUN curl https://s3.amazonaws.com/cfpb-hmda-public/prod/cypress/fixtures.zip > ./cypress/downloads/fixtures.zip
+RUN unzip -o cypress/downloads/fixtures.zip -d cypress/
+RUN curl https://s3.amazonaws.com/cfpb-hmda-public/prod/cypress/e2e.zip > ./cypress/downloads/e2e.zip 
+RUN unzip -o cypress/downloads/e2e.zip -d cypress/  
+RUN mkdir -p cypress/e2e/docusaurus
+RUN curl https://raw.githubusercontent.com/cfpb/hmda-combined-documentation/main/cypress/e2e/documentation.cy.js > cypress/e2e/docusaurus/documentation.cy.js
+
+## Remove specific modules with critical CVEs from cypress cache
+RUN rm -rf /root/.cache/Cypress/13.6.4/Cypress/resources/app/node_modules/loader-utils/
+RUN rm -rf /root/.cache/Cypress/13.6.4/Cypress/resources/app/node_modules/underscore/
+RUN rm -rf /root/.cache/Cypress/13.6.4/Cypress/resources/app/node_modules/mocha-7.0.1/node_modules/minimist/
+RUN rm -rf /root/.cache/Cypress/13.6.4/Cypress/resources/app/node_modules/flat/
+## find / -name execa > execalist
+## for f1 in $(cat execalist) ; do grep version $f1/package.json ; done
```

Comment:
Can we clean up the unnecessary commented out code?

Reply:
This will help figure our future CVEs. Cypress dependencies are difficult to track, this script will allow to find them and resolve